src/Controller/Admin/AddressCrudController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\Address;
+use EasyCorp\Bundle\EasyAdminBundle\Config\Crud;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Field\DateTimeField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\FormField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\NumberField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextField;
+
+class AddressCrudController extends AbstractCrudController
+{
+    public static function getEntityFqcn(): string
+    {
+        return Address::class;
+    }
+
+    public function configureCrud(Crud $crud): Crud
+    {
+        return $crud
+            ->setDefaultSort(['id' => 'DESC']);
+    }
+
+    public function configureFields(string $pageName): iterable
+    {
+        return [
+            IdField::new('id')
+                ->setDisabled()
+                ->hideWhenCreating(),
+
+            FormField::addFieldset('Address'),
```

Comment:
Make labels and other static text translatable:

```suggestion
            FormField::addFieldset(new TranslatableMessage('Address')),
```

https://symfony.com/doc/current/translation.html#translatable-objects

Reply:
Fixed